raygun4go.go

```diff
@@ -234,6 +235,19 @@ func (c *Client) CreateError(message string) error {
 	return c.Submit(post)
 }
 
+// Manually send an error to Raygun with a custom message and a custom stack trace.
+// This is particularly useful in scenarios where the default system stack trace is not relevant or sufficient.
+//
+// message: The error message to be reported.
+// st:      A custom stack trace to be associated with the error.
+// Users can manually construct a StackTrace using the AddEntry method.
```

Comment:
Wondering if we want to mention where that AddEntry method is - or have an example of how to use it. The way it currently reads sounds like AddEntry is on this Raygun client.

Reply:
+1 
it would be useful if the actual stack trace type was documented and referenced in this method